bleak/backends/scanner.py

```diff
@@ -81,6 +81,13 @@ class BaseBleakScanner(abc.ABC):
             containing this advertising data will be received.
     """
 
+    seen_devices: Dict[str, Tuple[BLEDevice, AdvertisementData]]
+    """
+    Map of device identifier to BLEDevice and most recent advertisement data.
+
+    This map must be cleared when scanning starts.
+    """
+
```

Comment:
What is the point of this class variable in addition to the instance variable defined below?